cmd/neofs-node/config/engine/shard/blobstor/fstree/config.go

```diff
@@ -38,3 +38,10 @@ func (x *Config) Depth() uint64 {
 
 	return DepthDefault
 }
+
+// NoSync returns the value of "no_sync" config parameter.
+//
+// Returns false if the value is not a boolean or is missing.
+func (x *Config) NoSync() bool {
```

Comment:
Lets add to `config/example` and cover in corresponding unit-test?

Reply:
Fixed